src/Extension/Events/QuestionTextEvaluationProduced.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.DotNet.Interactive.Commands;
+
+namespace Extension.Events
+{
+    public class QuestionTextEvaluationProduced : EvaluationProduced
```

Comment:
What is the goal of using inheritance here?

Reply:
Same comments as above